### DISCUSSION TITLE
How to mask AutoFiled?

### DISCUSSION THREAD

**Discussion by alex09j22** [vazco/uniforms#1034] [Q&A] [answered]
How to mask AutoField?
Ideally I'd love to use react-input-mask package.

**Comment by Monteth** [comment, marked as answer]
Hello @mingcnjs, it seems like you are looking for [overriding AutoField](https://uniforms.tools/docs/uth-autofield-algorithm/#overriding-autofield). You can create a custom field using `react-input-mask` and render it conditionally basing on the field props.
